README.md

```diff
@@ -403,109 +422,133 @@ $ espercli group unset-active
-name          4G
-device_count  2
+TITLE           DETAILS
+id              5fd4e150-7a91-4d73-a379-cac0795bd949
+name            TestB
+parent_id       bb6be630-a2fc-4b57-bafa-fdda92433684
+device_count    0
+path            All devices/TestB/TestBB
+children_count  0
 ```
 
 #### 7. delete
-Remove particular group.
+Remove particular group. Here, if `group-name` is given, the first group from the response list with the given name will be removed. If `--groupid` or `-id` option and `group-name` is given, the corresponding group will be removed.
 ```sh
-$ espercli group delete [group-name]
+$ espercli group delete [OPTIONS] [group-name] 
 ```
+##### Options
+| Name, shorthand | Default| Description|
+| -------------   |:------:|:----------|
+| --groupid, -id  |        | Group id  |
 
 ##### Example
 ```sh
-$ espercli group delete 5G
-Group with name 5G deleted successfully
+$ espercli group delete -id 69f5f7ac-c182-4f31-a6bc-f895bdc44a70 Test-BB
+Group with name Test-BB deleted successfully
 ```
 
 #### 8. add
-Add devices into a group, active group is used to add devices if `--group` or `-g` option is not given explicitly.
+Add devices into a group, active group is used to add devices if `--group` or `-g` option is not given explicitly. A maximum of 1000 devices can be added at a time.
+Here, if `--group` or `-g` is given, devices will be added to the first group from the response list with the given name. If `--groupid` or `-id` is also given, then devices will be added to corresponding group.
 ```sh
 $ espercli group add [OPTIONS]
 ```
 ##### Options
 | Name, shorthand | Default| Description|
 | -------------   |:------:|:----------|
 | --group, -g     |        | Group name |
+| --groupid, -id  |        | Group id   |
 | --devices, -d   |        | List of device names, list format is space separated |
 | --json, -j      |        | Render result in JSON format |
 
 ##### Example
 ```sh
-$ espercli group add -g 5G -d SNA-SNL-73YE SNA-SNL-NYWL 
+$ espercli group add -g TestBB -id bb6be630-a2fc-4b57-bafa-fdda92433684 -d DEV-ELOP-UULA
 
-TITLE         DETAILS
-id            2e5efca2-7776-442e-a5ef-c2758d4a45a3
-name          5G
-device_count  2
+TITLE           DETAILS
+id              bb6be630-a2fc-4b57-bafa-fdda92433684
+name            TestBB
+parent_id       685484fa-eb6a-4ef9-a0f1-63f6febf7ce3
+device_count    1
+path            All devices/TestBB
+children_count  1
 ```
 
 #### 9. remove
-Remove devices from a group, active group is used to add devices if `--group` or `-g` option is not given explicitly.
+Remove devices from a group, active group is used to add devices if `--group` or `-g` option is not given explicitly. A maximum of 1000 devices can be removed at a time.
+Here, if `--group` or `-g` is given, devices will be removed from the first group from the response list with the given name. If `--groupid` or `-id` is also given, then devices will be removed from the corresponding group.
```

Comment:
Pls add like when devices are removing from a group, those will be added to an immediate parent group